docs/how-to-guides/13_how-to-propose-a-multisig-transaction.md

```diff
@@ -0,0 +1,187 @@
+```javascript
+const proposeInput = {
+    proposer: 'useraaaaaaaa',
+    proposal_name: 'changeowner',
+    requested: [
+      {
+        actor: 'useraaaaaaaa',
+        permission: 'active'
+      },
+      {
+        actor: 'userbbbbbbbb',
+        permission: 'active'
+      }
+    ],
+    trx: {
+      expiration: '2019-09-14T16:39:15',
+      ref_block_num: 0,
+      ref_block_prefix: 0,
+      max_net_usage_words: 0,
+      max_cpu_usage_ms: 0,
+      delay_sec: 0,
+      context_free_actions: [],
+      actions: serialized_actions,
+      transaction_extensions: []
+    }
+  };
+```
+
+## Propose
+In the example below, a transaction is submitted to the `propose` action of the `eosio.msig` contract using the `proposeInput` object created [above](#propose-input).
+```javascript
+await api.transact({
+  actions: [{
+    account: 'eosio.msig',
+    name: 'propose',
+    authorization: [{
+      actor: 'useraaaaaaaa',
+      permission: 'active',
+    }],
+    data: proposeInput,
+  }]
+}, {
+  blocksBehind: 3,
+  expireSeconds: 30,
+  broadcast: true,
+  sign: true
+});
+```
+
+## Propose a Multi-sig Transaction
+Below all three steps to propose a multi-sig transaction are provided.
+
+```javascript
+// CREATE ACTION TO PROPOSE
+const actions = [
+  {
+    account: 'eosio',
+    name: 'updateauth',
+    authorization: [
+      { 
+        actor: 'useraaaaaaaa',
+        permission: 'active',
+      }
+    ], data: {
+      account: 'useraaaaaaaa',
+      permission: 'active',
+      parent: '',
+      auth: {
+        threshold: 1,
+        keys: [
+          {
+            key: 'PUB_R1_6FPFZqw5ahYrR9jD96yDbbDNTdKtNqRbze6oTDLntrsANgQKZu',
+            weight: 1
+          }
+        ],
+        accounts:[],
+        waits:[]
+      }
+    },
+  }
+];
+
+(async () => {
+  const serialized_actions = await api.serializeActions(actions)
+
+  // BUILD THE MULTISIG PROPOSE TRANSACTION
+  proposeInput = {
+    proposer: 'useraaaaaaaa',
+    proposal_name: 'changeowner',
+    requested: [
+      {
+        actor: 'useraaaaaaaa',
+        permission: 'active'
+      },
+      {
+        actor: 'userbbbbbbbb',
+        permission: 'active'
+      }
+    ],
+    trx: {
+      expiration: '2019-09-16T16:39:15',
+      ref_block_num: 0,
+      ref_block_prefix: 0,
+      max_net_usage_words: 0,
+      max_cpu_usage_ms: 0,
+      delay_sec: 0,
+      context_free_actions: [],
+      actions: serialized_actions,
+      transaction_extensions: []
+    }
+  };
+
+  //PROPOSE THE TRANSACTION
+  const result = await api.transact({
+    actions: [{
+      account: 'eosio.msig',
+      name: 'propose',
+      authorization: [{
+        actor: 'useraaaaaaaa',
+        permission: 'active',
+      }],
+      data: proposeInput,
+    }]
+  }, {
+    blocksBehind: 3,
+    expireSeconds: 30,
+    broadcast: true,
+    sign: true
+  });
+})();
+```
```

Comment:
This is awesome.  Really clear.  Simple.  and answering a question the community has asked...